caikit/core/data_model/base.py

```diff
@@ -933,7 +937,7 @@ def _default_serialization_overrides(obj):
             - datetime.datetime
             """
             if isinstance(obj, bytes):
-                return base64.encodebytes(obj).decode("utf-8")
+                return base64.b64encode(obj).decode("utf-8")
```

Comment:
Interesting. What exactly does this change do? I've never fully known the difference between these two methods

Reply:
Initially I thought `encodebytes` was just the legacy interface to `b64encode` but it turns out that [encodebytes](https://docs.python.org/3/library/base64.html#base64.encodebytes) adds new line characters every 76 bytes which messed with tests. I'm not sure it actually matters though as parsers/validators should still handle it